core/files/css/base.php

```diff
@@ -673,6 +678,38 @@ protected function add_control_style_rules( array $control, array $values, array
 		);
 	}
 
+	/**
+	 * Get Control Global Default Value
+	 *
+	 * If the control has a global default value, and the corresponding global default setting is enabled, this method
+	 * fetches and returns the global default value. Otherwise, it returns null.
+	 *
+	 * @since 3.7.0
+	 * @access private
+	 *
+	 * @param $control
+	 * @return string|null
+	 */
+	private function get_control_global_default_value( $control ) {
+		if ( ! empty( $control['global']['default'] ) ) {
```

Comment:
Try `if ( empty( $control['global']['default'] ) ) {` and exit from the function early..

Reply:
Done